target/linux/ath79/image/common-yuncore.mk

```diff
@@ -0,0 +1,11 @@
+define Build/yuncore-tftp-header-qca953x
+	{ \
+		echo -n -e "YUNCOREsetenv bootcmd \"bootm 0x9f050000 || bootm 0x9fe80000\" &&" \
+			"setenv resetcmd \"erase 0x9f050000 +0xfa0000 && cp.b 0x800600c0 0x9f050000 0xfa0000\" && \
+			"setenv bootargs \"console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd\" && \
+			"saveenv && erase 0x9f050000 +0xfa0000 && cp.b 0x800600c0 0x9f050000 0xfa0000" | \
```

Comment:
WRONG. And please don't include code from my PR here.

Reply:
well, I am pissed of with al this negativity here

If you want do it in another PR, then please do it properly, something that DOES work, not something you THINK it works

Reply:
DON'T USE MY CODE HERE AND WRONG!